.github/actions/verify-token/package.json

```diff
@@ -26,8 +26,7 @@
     "@actions/github": "^5.1.1",
     "@octokit/webhooks-types": "^6.9.0",
     "sigstore": "^1.1.1",
-    "tscommon": "file:../tscommon/tscommon-0.1.0.tgz",
-    "yaml": "^2.2.1"
```

Comment:
Did we not need the `yaml` package anymore?

Reply:
I think it was never needed. Pre-submit are passing. tscommon only uses files and crypto deps.

Reply:
Sorry, mis-read. I thought the dep was imported in the tscommon. Why did the pre-submit not catch it? Weird, ill send a PR

Reply:
Yeah, I think it works since all the tests run `npm ci` which just looks at `package-lock.json`. Since it's included there everything's fine.

keeping `package.json` in sync with `package-lock.json` is mostly for us developers to make sense of the dependencies.

Reply:
ah, makes sense.